### PR TITLE
Update PubbyStation

### PR DESCRIPTION
**Pull request by pubby**
A bunch of changes and several fixes.

Some pictures are in this thread:
https://tgstation13.org/phpBB/viewtopic.php?p=395721#p395721

:cl: Pubby
tweak: PubbyStation has been updated. You can now be the lawyer.
/:cl:

### CODE REVIEW

**Comment by 81Denton** [comment]
That is one sexy shuttle

File: _maps/shuttles/emergency_pubby.dmm [hunk 1]
```diff
@@ -42,94 +35,93 @@
...
+/obj/machinery/computer/security/telescreen/entertainment{
+	pixel_y = 32
+	},
+/obj/item/twohanded/required/kirbyplants{
+	icon_state = "plant-21";
+	pixel_x = -3;
+	pixel_y = 3
+	},
+/turf/open/floor/plasteel/darkgreen/side{
+	tag = "icon-darkgreen (NORTHEAST)";
```
Comment (ShizCalev) [inline _maps/shuttles/emergency_pubby.dmm]: Dv

File: _maps/shuttles/emergency_pubby.dmm [hunk 2]
```diff
@@ -42,94 +35,93 @@
...
+	icon_state = "darkgreen";
```
Comment (ShizCalev) [inline _maps/shuttles/emergency_pubby.dmm]: Nope

File: _maps/shuttles/emergency_pubby.dmm [hunk 3]
```diff
@@ -42,94 +35,93 @@
...
-	pixel_y = -6
-	},
-/obj/machinery/light/small{
-	brightness = 3;
+/obj/structure/chair/comfy{
 	dir = 8
 	},
-/turf/open/floor/mineral/plastitanium/brig,
+/turf/open/floor/plasteel/darkgreen/side{
+	icon_state = "darkgreen";
```
Comment (ShizCalev) [inline _maps/shuttles/emergency_pubby.dmm]: Dv

File: _maps/shuttles/emergency_pubby.dmm [hunk 4]
```diff
@@ -141,493 +133,832 @@
...
 /area/shuttle/escape)
 "aD" = (
-/obj/machinery/door/airlock/titanium{
-	name = "Emergency Shuttle Airlock";
-	req_access_txt = "2"
+/obj/structure/chair/comfy{
+	dir = 1
+	},
+/turf/open/floor/plasteel/darkgreen/side{
+	tag = "icon-darkgreen (SOUTHWEST)";
```
Comment (ShizCalev) [inline _maps/shuttles/emergency_pubby.dmm]: Anything that's tag = "icon is a dv

I'd mark them all but I'm on my phone till next week.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by pubby** [comment]
I think it's good now?

**Comment by pubby** [comment]
Cyberboss your render robot is having difficulties.

I would like this merged soon so that I can do map changes in #37833

**Pull request merged**